src/components/grids/PlanGrid.tsx

```diff
@@ -409,6 +419,14 @@ const PlanGrid = () => {
         >
           <IconRotate2 stroke='2.5' size={20} />
         </ActionIcon>
+        <ActionIcon
```

Comment:
The rotate buttons look a bit strange now that there are two buttons in between them. Could you maybe rearrange them to go in this order: rotate left, rotate right, duplicate, delete?

Reply:
Done